1010.1819/1010.1819.yaml

```diff
@@ -154,14 +160,16 @@
       score: 
         Haining: 1
         Will: 
-        Yasaman: 
+        Yasaman: 0
+        # deducted due to too much irrelevant information.
```

Comment:
Agree, it contains too much info.. not a valid answer. 
I will come back on this later.